DOCKER.md

```diff
@@ -16,15 +18,32 @@ to avoid using stale layers
 
 The optional `JENKINS_VERSION` specifies the version of Jenkins core.
 
+Notice that the master workspace is mapped to `/build`.
+This directory can be exposed as a volume.
+
+
 ## Usage
 Run the image by providing path to `Jenkinsfile` . For example,
 
 ```bash
 docker run --rm -v $PWD/test:/workspace jenkinsfile-runner:my-production-jenkins
 ```
 
+In same cases you can be interested in passing extra options.
+It can be done passing the `JAVA_OPTS` environment variable.
+
+```bash
+docker run --rm -e JAVA_OPTS="-Xms 256m" -v $PWD/test:/workspace jenkinsfile-runner:my-production-jenkins
+```
+
+It is also possible to pass arguments to the Jenkinsfile or to execute it outside a sandbox environment with the `-ns` and `-a` options.
```

Comment:
This line is a bit confusing. The example below clarifies it tho.
I would also recommend to put a warning that non-sandbox mode may pose potential security risks, and hence it should be used carefully

Reply:
Done!